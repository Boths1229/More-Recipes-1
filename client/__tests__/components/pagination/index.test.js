import React from 'react';
import { shallow } from 'enzyme';
import Paginate from '../../../components/pagination/index';


let props;

const setup = () => {
  props = {
    handlePaginationChange: () => {},
    page: 2,
  };

  return shallow(<Paginate {...props} />);
};

describe('Pagination Component', () => {
  it('should render correctly', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
