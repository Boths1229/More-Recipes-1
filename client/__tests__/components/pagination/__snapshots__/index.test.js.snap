// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination Component should render correctly 1`] = `
<div
  className="container"
>
  <PaginationBoxView
    activeClassName="active"
    breakClassName="page-link"
    breakLabel={
      <a
        href=""
      >
        ...
      </a>
    }
    containerClassName="pagination custom-pagination"
    disableInitialCallback={false}
    disabledClassName="disabled"
    marginPagesDisplayed={3}
    nextClassName="page-item"
    nextLabel="Next"
    nextLinkClassName="page-link custom-link"
    onPageChange={[Function]}
    pageClassName="page-item"
    pageCount={2}
    pageLinkClassName="page-link custom-link"
    pageRangeDisplayed={2}
    previousClassName="page-item"
    previousLabel="Previous"
    previousLinkClassName="page-link custom-link"
    subContainerClassName="pages pagination"
  />
</div>
`;
